File: Iniciantes/b1114.java
import java.util.Scanner;

public class b1114 {
	public static void main(String[] args) throws Exception {
		Scanner sc = new Scanner(System.in);

		int senha = sc.nextInt();

		while (senha != 2002) {
			System.out.println("Senha Invalida");
			senha = sc.nextInt();
		}

		System.out.println("Acesso Permitido");
	}
}

File: Iniciantes/b1114.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

const senhaValida = 2002

let senha = parseInt(lines.shift());

while(senha != senhaValida){
  console.log('Senha Invalida');
  senha = parseInt(lines.shift());
}

console.log('Acesso Permitido');